### ISSUE TITLE
Better result with Normal vo?

### ISSUE THREAD

**Issue by xlong0513** [syinari0123/SuperPoint-VO#4]
Hi, thanks for the code.
I have run the code with 00 sequences of kitti, but i cannot get the same results you provided in repository. In contrast, normal VO seems better than Super-VO, as follows
![map](https://user-images.githubusercontent.com/35829321/70025160-2a4c0b00-15d7-11ea-8768-a88ca17fc9a9.png)

Can u tell us what the problem is? Thanks

**Comment by frspring** [comment]
I don't think the author test the result in a right way.It's obvious that the trajectory is not aligned.

**Comment by xlong0513** [comment]
@frspring Right, especially the results on kitti_00. If so, the SuperPoint is worse than the normal method in VO.
